### ISSUE TITLE
FileNotFoundException

### ISSUE THREAD

**Issue by eleventeen** [freecode/YouTube-Downloader#2]
https://www.youtube.com/watch?v=S_w72-AsoGQ - response doesn't contain "signature=". But it is added to urls list. So the first problem:
if (!u.startsWith("http") && !isValid(u)) {
should be
if (!u.startsWith("http") || !isValid(u)) {
Response does containt "s=" with value that looks similar to the signature from other links. But after replacing "s=" with "signature=" still error code 403 is returned.

**Comment by l3eta** [comment]
Yeah I ran into that after using it so long. What sparked people using this
anyways
